hedera-node/src/main/java/com/hedera/services/txns/schedule/ScheduleProcessing.java

```diff
@@ -0,0 +1,288 @@
+
+			if (!seen.add(nextLong)) {
+				log.error("tried to process the same transaction twice! {}", next);
+				throw new IllegalStateException("tried to process the same transaction twice!");
+			}
+
+			// if we were going to check throttling to prevent scheduled transactions from all executing
+			// rapidly after downtime, it would be done here. We don't do that currently.
+
+			try {
+
+				if (!this.dynamicProperties.schedulingLongTermEnabled()) {
+					// if long term is disabled, we always expire transactions that would otherwise
+					// execute autonomously
+					store.expire(next);
+					sigImpactHistorian.markEntityChanged(nextLong);
+					continue;
+				}
+
+				var schedule = store.get(next);
+
+				if (!this.isFullySigned.test(schedule)) {
+
+					// expire transactions that are not ready to execute
+					store.expire(next);
+					sigImpactHistorian.markEntityChanged(nextLong);
+
+				} else if (onlyExpire) {
+
+					// if we are only expiring, we have to stop processing here and return null
+					return null;
+
+				} else {
+
+					var triggerResult = scheduleExecutor.getTriggeredTxnAccessor(next, store, false);
+
+					if (triggerResult.getLeft() != OK) {
+						log.error("Scheduled transaction was not trigger-able even though it should be! Expiring it. {}",
+								next);
+						store.expire(next);
+						sigImpactHistorian.markEntityChanged(nextLong);
+					} else {
+						return triggerResult.getRight();
+					}
+
+				}
+			} catch (Exception e) {
+				log.error("SCHEDULED TRANSACTION SKIPPED!! Failed to triggered transaction due unexpected error! {}",
+						next, e);
+
+				// Immediately expire malfunctioning transactions, if we get here then there is a bug.
+				// We can't leave it in the db, it will prevent other scheduled transactions from processing.
+				store.expire(next);
+				sigImpactHistorian.markEntityChanged(nextLong);
+			}
+
+		}
+
+		log.error("Reached maxProcessingLoopIterations reached in triggerNextTransactionExpiringAsNeeded, we should never get here!");
+
+		return null;
+	}
+
+	/**
+	 * @param schedule a schedule to check the "future throttles" for.
+	 * @return an error code if there was an error, OK otherwise
+	 */
+	public ResponseCodeEnum checkFutureThrottlesForCreate(final ScheduleVirtualValue schedule) {
+
+		if (dynamicProperties.schedulingLongTermEnabled()) {
+			scheduleThrottling.resetUsage();
+
+			final TreeMap<RichInstant, List<TxnAccessor>> transactions = new TreeMap<>();
+
+			var curSecond = schedule.calculatedExpirationTime().getSeconds();
+
+			var bySecond = store.getBySecond(curSecond);
+
+			if (bySecond != null) {
+				bySecond.getIds().values().forEach(ids -> ids.forEach(id -> {
+					var existing = store.getNoError(EntityNum.fromLong(id).toGrpcScheduleId());
+
+					if (existing != null) {
+						if (existing.calculatedExpirationTime().getSeconds() != curSecond) {
+							log.error("bySecond contained a schedule in the wrong spot! Ignoring it! spot={}, id={}, schedule={}",
```

Comment:
We try to reserve `ERROR` for unrecoverable events; here a `WARN` seems more appropriate, since we just skip the misplaced schedule.

Reply:
Changing. Are the other logs here and in schedulestore at the appropriate level? I think the error ones really would be bad if they are happening.